### ISSUE TITLE
decimal data type convertion

### ISSUE THREAD

**Issue by trrodrigues** [go-pg/pg#836]
I think that decimal/numeric postgresql data type are converted to float64. Why? 

Can I create a custom convertion to a custom decimal type?

**Comment by vmihailenco** [comment]
Can you provide some code that demonstrates mentioned conversion?

**Comment by trrodrigues** [comment]
I'm checking that in go-pg tests.

https://github.com/go-pg/pg/blob/2f7c67c51832243161ccbcfea5851f9111d95ca6/orm/create_table_test.go#L23

It's possible to create a custom conversion to a custom decimal type?

**Comment by vmihailenco** [comment]
Yes, using database/sql interfaces to define custom. Or you can use existing one https://github.com/shopspring/decimal

```
Decimal decimal.Decimal `sql:"type:'decimal(10,10)'"`
```
